examples/diagnostics_and_criticism/sampler-stats.ipynb

```diff
@@ -7,11 +7,6 @@
     "(sampler_stats)=\n",
```

Comment:
<p>Is there a "keyword" way to reference/link to this bit of documentation?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>